mod/admin.php

```diff
@@ -917,6 +917,7 @@ function update_table($table_name, $fields, $old_url, $new_url)
 			$upds = implode(", ", $upd);
 
 			$r = q("UPDATE %s SET %s;", $table_name, $upds);
+
```

Comment:
I personally like how the `$r` variable is check immediately after the assignation, especially since we don't use it afterwards. In reality this could be `if (!DBM::is_result(q("UPDATE %s SET %s;", $table_name, $upds))) {` but for ease of reading we broke it down with an single-use intermediate variable.

A new line seems to imply `$r` is more important than it actually is.

Reply:
I have added it for better readability that the code is lesser compacted.

Reply:
Sure, it isn't a problem if I can't convince you to revert the change, it's perfectly valid.